src/test/java/no/nav/familie/integrasjoner/arbeidoginntekt/ArbeidOgInntektTestConfig.kt

```diff
@@ -0,0 +1,22 @@
+package no.nav.familie.integrasjoner.arbeidoginntekt
+
+import io.mockk.every
+import io.mockk.mockk
+import no.nav.familie.integrasjoner.client.rest.ArbeidOgInntektClient
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Primary
+import org.springframework.context.annotation.Profile
+
+@Configuration
+class ArbeidOgInntektTestConfig {
+    @Bean
+    @Profile("mock-arbeid-og-inntekt")
+    @Primary
+    @Throws(Exception::class)
+    fun clientMock(): ArbeidOgInntektClient {
+        val clientMock: ArbeidOgInntektClient = mockk(relaxed = true)
+        every { clientMock.hentUrlTilArbeidOgInntekt(any()) } returns "https://www.gyldig-url.no"
```

Comment:
Vet det er gjort overalt i koden våres, men dette patternet med mockk funker ikke så bra og er roten til at testene noen steder er ustabile. Blant annet i ba-sak. Faren er ganske liten i integrasjoner siden testene stort sett bruker wiremock og mer separert fra hverandre. Grunnen er at mellom hver test så vil mockk resette mocks. Da forsvinner denne mocken. For å unngå dette så har man satt på en annotation som heter noe ala KeepMocks, men hvis en test endrer oppførsel på mocken så vil testen ikke lenger være uavhengig av hverandre.